src/Hook/Feature.php

```diff
@@ -141,12 +199,75 @@ public function actionFeatureSave(array $params)
                 'INSERT INTO ' . _DB_PREFIX_ . 'layered_indexable_feature_lang_value
                 (`id_feature`, `id_lang`, `url_name`, `meta_title`)
                 VALUES (
-                ' . (int) $params['id_feature'] . ', ' . (int) $language['id_lang'] . ',
+                ' . $featureId . ', ' . (int) $language['id_lang'] . ',
                 \'' . pSQL(Tools::link_rewrite($seoUrl)) . '\',
                 \'' . pSQL($metaTitle, true) . '\')'
             );
         }
 
         $this->module->invalidateLayeredFilterBlockCache();
     }
+
+    /**
+     * Saves feature form. @since ps 1.7.7
+     *
+     * @param array $params
+     */
+    private function save(array $params)
```

Comment:
Code is duplicate from displayFeatureForm, I'm not in favor to have duplicate sql queries.

Reply:
I guess You ment `actionFeatureSave`, removed duplication from there.